README.md

```diff
@@ -40,6 +40,61 @@ If you're interested in being a validator on the Aion networks, refer to our [Va
 ### Users
 If you're interested in interacting with dApps and _using_ Aion, refer to our [Aion Desktop Wallet Docs](https://docs-aion.theoan.com/docs/wallets)
 
+## FAQ
+1. Where can I store my Aion?
+We recommend using either the desktop wallet or the web-based Aion Wallet; more information can be found in “Docs” (link at top of page).
```

Comment:
The desktop Wallet is deprecated, it might be better to only sugest for the web base